src/components/faq/helpers/utils.ts

```diff
@@ -49,3 +49,14 @@ export function filterFaqQuestions(
 
   return filteredFaqQuestions
 }
+
+export const dateToTime = (dateString: string | undefined) => {
+  if (!dateString) {
+    return
+  }
+
+  const date = new Date(dateString)
+  const localTime = date.toLocaleTimeString()
+
+  return localTime
+}
```

Comment:
There is a `formatDateString` function in `src/common/util/date.ts` you can use

Reply:
Unfortunately, this function doesn't meet my needs

Reply:
Maybe it would make more sense to have this function to there then.